src/headless/converse-muc.js

```diff
@@ -507,6 +507,40 @@ converse.plugins.add('converse-muc', {
                     }, null, 'message', 'groupchat', null, room_jid,
                     {'matchBareFromJid': true}
                 );
+                this.muc_notifications_handler = _converse.connection.addHandler(stanza => {
+                    const item = converse_muc_sizzle("x[xmlns=\"".concat(converse_muc_Strophe.NS.MUC_USER, "\"] item"), stanza).pop();
+
+                    if (item) {
+                        const from = stanza.getAttribute("from");
+                        const type = stanza.getAttribute("type");
+                        const affiliation = item.getAttribute('affiliation');
+                        const jid = item.getAttribute('jid');
+
+                        const data = {
+                          'from': from,
+                          'nick': converse_muc_Strophe.getNodeFromJid(jid),
+                          'type': type,
+                          'states': [],
+                          'show': type == 'unavailable' ? 'offline' : 'online',
+                          'affiliation': affiliation,
+                          'role': item.getAttribute('role'),
+                          'jid': jid
+                        }
+
+                        console.debug("muc_handler - muc occupant push", data);
```

Comment:
Don't use the `console` directly, if you want to log information, then import and use the `log` utility. There are lots of examples in the code of how this is done.